.gitignore

```
@@ -25,6 +25,7 @@
 *.la
 *.a
 *.lib
+!lib/**/*.a
```

Hvis det bare er for din custom libjsoncpp så vil jeg hellere at det kun er den specifikt der bliver tracked.
```suggestion
!lib/**/libjsoncpp.a
```